dvc/repo/experiments/remove.py

```diff
@@ -33,6 +49,33 @@ def remove(repo, exp_names=None, queue=False, **kwargs):
     return removed
 
 
+def _clear_workspace_stash(repo):
```

Comment:
```suggestion
def _clear_stash(repo):
```

This clears the entire stash, it is not specific to stashed experiments which only apply to the current workspace/HEAD